### DISCUSSION TITLE
Status of users when they have finished their trial but have not yet subscribed

### DISCUSSION THREAD

**Discussion by FlashInThePan** [Glench/ExtPay#111] [General]
Hey guys, 

Could I just confirm what the extpay.js will return for a user who HAS completed their trial but has NOT jet subscribed please.. Specifically, will the following conditions all be true?

(user.trialStartedAt is not null) and (there has been over 7 days since user.trialStartedAt) and (user.subscriptionStatus is "unpaid")

Thanks in advance!

**Comment by Glench** [comment]
Everything is right except user.subscriptionStatus will not be present or will be null. It won't have status unpaid (which is a Stripe status meaning they've paid in the past).

**Reply by FlashInThePan** [reply]
Perfect. Thanks!
